pkg/trace/obfuscate/sql_tokenizer.go

```diff
@@ -559,7 +559,7 @@ func (tkn *SQLTokenizer) scanBindVar() (TokenKind, []byte) {
 		token = ListArg
 		tkn.advance()
 	}
-	if !isLetter(tkn.lastChar) {
+	if !isLetter(tkn.lastChar) && !unicode.IsSpace(tkn.lastChar) {
```

Comment:
Technically, the token is neither a `ValueArg` nor `ListArg`; it is simply `':'`. If we end up applying a rule later which processes this token, then we should return the correct token.

Reply:
ah yes good point, let me fix that real quick